site/src/Controller/OrganizationController.php

```diff
@@ -215,6 +220,20 @@ public function joinAction($organizationName, OrganizationRepository $repository
         $member->setUser($user);
         $member->setPermission(OrganizationMember::PERMISSION_REQUEST);
         $em = $this->getDoctrine()->getManager();
+        // Find Organisation members
+        $orgAdmin = $repository->findOneByName($organizationName)->getCreator();
+        // Queue notifications to the organisation admin
+        $notification = new Notification();
+        $notification->setType("organisation_join_request")
+            ->setSubject("Organisation join request")
+            ->setRecipient($orgAdmin)
+            ->setMessage(
+                $member->getUser()->getUsername()." has requested to join ".
+                $organizationName . ". Kindly go to https://www.librecores.org/"
```

Comment:
Please use the router component to construct this URL, and ensure that the domain is not hardcoded but depends on the configuration (or in this case, on the request).

Reply:
+1